aries-site/src/examples/templates/filtering/FilteringCards.js

```diff
@@ -254,6 +256,7 @@ const Filters = ({
     setName([]);
     setFilters(defaultFilters);
     setFiltering(false);
+    setSearch();
```

Comment:
Since we default to `''` should this `setSearch('')` ?

Reply:
Yes, good catch!

Reply:
Yes fixing now!